components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.cpp

```diff
@@ -10,7 +12,7 @@ namespace scream
 Nudging::Nudging (const ekat::Comm& comm, const ekat::ParameterList& params)
   : AtmosphereProcess(comm, params)
 {
-  m_datafiles  = m_params.get<std::vector<std::string>>("nudging_filename");
+  m_datafiles  = filename_glob(m_params.get<std::vector<std::string>>("nudging_filename"));
```

Comment:
This may be personal, but since we use patterns, we should probably change the param name to something like `nudging_filenames_patterns`? The param name, as of now, makes it sound like you must pass actual filenames (and it's not even clear on the fact that 2+ are allowed).

Reply:
Thanks Luca, I just didn't intend to change any existing user-level parameters... the user should know that 2+ strings are allowed for nudging_filename as shown in the array type specified in its definition? How about just adding a doc to indicate that nudging_filename can accept patterns in namelist_defaults_scream.xml? something like "one or more strings to indicate the actual filenames or patterns"?

Reply:
I agree that the user can see that they can pass more than 1 string, but the more pressing thing is that it's not very clear that these strings can be patterns, rather than actual file names.

I don't think it's a bad idea to change a parameter name and make it more clear. Imho, it will pay off in the long run.

Reply:
i see i'll change all nudging_filename to nudging_filenames_patterns